README.md

```diff
@@ -345,39 +263,39 @@ const sendEmail = msg.send("mailto://alice@example.com", {
 
 ## 3.1 Schema
 
-```ipldsch
+```
 type Task struct {
-  on    URI
-  call  Ability
+  rsc   URI
+  op    Ability
   input {String : Any}
-  nnc   string
+  nnc   String 
```

Comment:
Nonce at the level of Instruction makes them less reusable in `await`s.
They do make Instructions a more unique, and may be thought to be used in order for an Instruction to await some particular Receipt of other Instruction.
E.g., in Effects that depend on Result of the Receipt that caused them.
However, multiple Reciepts for the same Nonced Instruction still may exist, so it can't be used to uniquely identify a particular Receipt.

Perhaps the problem of "resolve await to this particular Receipt" could be addressed at the level of Task.
[Effects could be Instructions](https://github.com/ucan-wg/invocation/pull/21/files#r1316855451), and Tasks created for them could capture exact Receipt "awaited".
I.e., Tasks would become ala closures.
It's been previously proposed to capture "awaited" at the level of Recipes #22, some reasons for that listed there, perhaps doing at Task/Closure level is more ellegant.
```json
{
 "bafy...someInstruction":
 {"input": {"someAwaitedVal": {"await/ok": {"/": "bafy...instructionToAwait"}}}},

 "bafy...someInstructionTask":
 {"run": {"/": "bafy...someInstruction"},
  "awaited": [{"/": "bafy...instructionToAwaitReceipt"}]}
}
 ```

Reply:
> Nonce at the level of Instruction makes them less reusable for awaits.

Unique invocations are a hard requirement if you want to be able to schedule tasks easily, reuse results, and do semantic awaits. For deterministic wasm, the nonce should be set to just `""`, but te general case of effectful tasks need to have unique IDs per call, since they don't catpure their entire context in the invocation.